Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAutomationWorker.py

```diff
@@ -490,6 +487,12 @@ def read_omsconfig_file():
     if os.path.isfile(OMS_ADMIN_CONFIG_FILE):
         # the above path always points to the oms configuration file of the primary workspace
         keyvals = config_file_to_kv_pair(OMS_ADMIN_CONFIG_FILE)
+        if os.path.isfile(OMS_AGENTID_FILE):
```

Comment:
if the file doesn't exist, exception is thrown, error is logged and set method exits.